### ISSUE TITLE
Design: Users cannot easily access action buttons in container list once they scroll far enough down the screen

### ISSUE THREAD

**Issue by mairin** [containers/podman-desktop#2637]
TLDR; We need a UI design proposal to allow for the action buttons that apply to selected items to always be accessible despite scrolling down the page.

## The problem to solve

We have some action buttons that can apply to a container selection:
![Screenshot from 2023-05-25 11-25-42](https://github.com/containers/podman-desktop/assets/799683/a773bf93-179a-4625-bf92-abd0dec35e9a)

There is delete and podify right now. The issue is that when you have a lot of of containers and you're selecting amongst them, you scroll down and the buttons disappear: 

### Before scroll
![Screenshot from 2023-05-25 11-24-45](https://github.com/containers/podman-desktop/assets/799683/fd2a6f61-e403-417e-937b-5d6a3d0262b1)

### After scroll
![Screenshot from 2023-05-25 11-34-35](https://github.com/containers/podman-desktop/assets/799683/08f2028f-3304-4f5b-82f5-39878fd89f2e)

**Comment by deboer-tim** [comment]
The search bar stays fixed and only the table scrolls now, so this issue is fixed and can be closed.
